results/user-testing-results.md

```diff
@@ -0,0 +1,138 @@
+- **This study is not an attempt to determine JupyterLab’s compliance with any single accessibility standard.** These results are not an accessibility statement for JupyterLab, JupyterHub, or any related Jupyter projects.
+
+## Findings
+
+This section summarizes issues, participant feedback and other patterns that appeared throughout test sessions. Findings are split between Navigation, UI customization, and Notebook concerns.
+
+### Navigation
+
+- **The menu bar (File, Edit, and so on) was the most used [JupyterLab region](https://jupyterlab.readthedocs.io/en/latest/user/interface.html) other than working in the notebook.** It was also most participants’ first choice to search when a task was unfamiliar to them. The left sidebar was used second most. The notebook toolbar was used third most. No participants used the cell toolbar or status bar. No tasks required features that are by default in the right sidebar, so it should not be counted.
+- **JupyterLab’s default focus styling needs to be improved.** Based only on the sessions, no regions appear have a designated [focus state](https://www.w3.org/WAI/WCAG22/Understanding/focus-visible.html). Some regions (like the file browser) have hover states that can be triggered by the keyboard and functionally tell users where their keyboard focus is. This is blatantly inaccessible because it prevents keyboard users from being able to tell where they are at JupyterLab at any given moment.
+- Many participants had to work to find things within the notebook (cells, headings, and outputs were requested throughout the test). They often navigated past their goal and had to start again. They rarely used features JupyterLab has to support navigating notebooks, like the [table of contents](https://jupyterlab.readthedocs.io/en/latest/user/toc.html) or [find](https://jupyterlab.readthedocs.io/en/latest/user/interface.html#searching) features. Only one participant used the table of contents; this participant explicitly praised it.
+- **Allowing the browser to take precedence wherever possible matters.** Multiple participants had browser-level settings, like high contrast settings or customized focus indicators, to improve their experience on all web pages. Participants with browser-level settings wanted them to carry over to JupyterLab. No major issues of JupyterLab overriding these settings occurred during these sessions, but participants made a point to call out the importance and their relief that they could use the settings they needed during their session.
```

Comment:
I was too! 

I have a feeling this may be mixed depending on how the user defines their own settings and that we essentially got lucky with participants who happened to have something compatible. We also may not have tripped all the right settings, ie. there was not as much keyboard shortcut behavior to compare.